modules/web/src/app/wizard/step/cluster/component.ts

```diff
@@ -167,6 +167,9 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
   backupStorageLocationsList: BackupStorageLocation[];
   backupStorageLocationLabel: BSLListState = BSLListState.Ready;
   provider: NodeProvider;
+  isAllowedIPRangeSupported: boolean = NODEPORTS_IPRANGES_SUPPORTED_PROVIDERS.includes(
+    this._clusterSpecService.provider
+  );
```

Comment:
We should keep the method in this component because user can go back in the wizard and change the provider.

Reply:
My previous comment was on the edit-cluster component.

Reply:
oops :sweat_smile:  sorry for that